### PR TITLE
Iotbzh/for upstream/add h3ulcb

### PR DESCRIPTION
**Pull request by julien-massot**

### CODE REVIEW

File: soc/arm/renesas_rcar/gen3/CMakeLists.txt [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# Copyright (c) 2021 IoT.bzh
+#
```
Comment (aaillet) [inline soc/arm/renesas_rcar/gen3/CMakeLists.txt]: delete line 2

File: soc/arm/renesas_rcar/Kconfig.defconfig [hunk 2]
```diff
@@ -0,0 +1,3 @@
+# SPDX-License-Identifier: Apache-2.0
+
```
Comment (aaillet) [inline soc/arm/renesas_rcar/Kconfig.defconfig:3]: missing copyright

**Review by aaillet — CHANGES_REQUESTED** [review]

File: soc/arm/renesas_rcar/Kconfig.defconfig (hunk 2 above)
Comment (julien-massot) [reply soc/arm/renesas_rcar/Kconfig.defconfig:3]: thx

File: soc/arm/renesas_rcar/gen3/CMakeLists.txt (hunk 1 above)
Comment (julien-massot) [reply soc/arm/renesas_rcar/gen3/CMakeLists.txt]: thx

**Review by aaillet — APPROVED** [review]

**Pull request closed without merging**